src/IssueViz.Security.UnitTests/Commands/HotspotsToolWindowCommandTests.cs

```diff
@@ -66,8 +67,8 @@ public void Ctor_CommandAddedToMenu()
 
             commandService.Verify(x =>
                     x.AddCommand(It.Is((MenuCommand c) =>
-                        c.CommandID.Guid == HotspotsToolWindowCommand.CommandSet &&
-                        c.CommandID.ID == HotspotsToolWindowCommand.ViewToolWindowCommandId)),
+                        c.CommandID.Guid == Constants.CommandSetGuid &&
```

Comment:
Minor refactoring. All of the command constants are in a single class and not duplicated in other classes.